lib/tasks/csv_load.rake

```diff
@@ -90,6 +90,14 @@ namespace :csv_load do
     end
   end
 
+  desc "Scorched Earth" 
+  task :pk_reset => :environment do
+    ActiveRecord::Base.connection.tables.each do |t|
```

Comment:
Great way to reset the pk.  May have to change the name from scorched earth to somthing that follows convention